docs/targets/zendesk.md

```diff
@@ -0,0 +1,68 @@
+# Event target for Zendesk
```

Comment:
Please ensure consistent capitalizations.  Each word should be capitalized _except_ for prepositions.

Reply:
For my own culture, why would we capitalize words that are not proper nouns? I rarely see that kind of casing. For example, you'll not see that in the AWS docs, or Zendesk docs, Microsoft docs, or any other docs I could think of.

If that's really a rule we want to stick to we should create a ticket to update all other docs pages as well.

Reply:
I'm reading this as a title, and thus under the MLA/AP style definition of titles: https://style.mla.org/capitalization-of-titles/

In this case, I'd expect `Event Target for Zendesk`

Reply:
OK thanks for the explanation. Created #33